.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,8 +2,8 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'bug', 'Status: Validation'
-assignees: ''
+labels: 'bug'
```

Comment:
I believe the label is currently `Type: Bug` instead.  I wonder if _that_ is the reason things broke?

Reply:
it's not possible to debug on GH just by try-error but it was a good catch @mook-as ...